emqx/manifest.json

```
@@ -0,0 +1,57 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "fa40ec7e-e8f6-4c4b-a675-31716b23a9df",
+  "app_id": "emqx",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Emqx integration with openmetrics v2",
+    "title": "EMQX",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "EMQX Broker Metrics",
+        "image_url": "images/emqx-dashboard.png"
+      }
+    ]
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Category::Metrics",
+      "Category::IoT",
+      "Submitted Data Type::Metrics"
+    ]
+  },
+  "assets": {
+    "integration": {
+      "source_type_name": "emqx",
+      "configuration": {
+        "spec": "assets/configuration/spec.yaml"
+      },
+      "events": {
+        "creates_events": false
+      },
+      "metrics": {
+        "prefix": "emqx.",
+        "check": ["emqx.cluster.nodes_running"],
+        "metadata_path": "metadata.csv"
+      },
+      "service_checks": {
+        "metadata_path": "assets/service_checks.json"
+      }},
```

```suggestion
      },
```
This may be causing the validation issues